.travis.yml

```diff
@@ -2,12 +2,15 @@ language: r
 sudo: false
 cache: packages
 
+os:
+  - linux 
+  - osx 
+
 install:
 - Rscript -e 'install.packages(c("Rcpp","testthat","covr","tractor.base","oro.nifti"))'
 
-script:
+after_script:
```

Comment:
This makes sense on the face of it, but [failures in `after_script` don’t break the build](https://docs.travis-ci.com/user/job-lifecycle#breaking-the-build), which is undesirable because I want to know when the standalone version stops working.